README.md

```diff
@@ -1,94 +1,130 @@
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=psidk ..
 make -j $(nproc)
 ctest -j $(nproc)
 make install
 ```
 
-The built product lives in `/opt/psidk`.
+The built product lives in `CMAKE_INSTALL_PREFIX` (`$CMAKE_BINARY_DIR/psidk` in the above example).
 
-To build documentation, use `-DBUILD_DOC=yes`
+The technical documentation for Psibase is part of this repository. To build & view it through the doc-sys applet of a deployed Psibase instance, see the instructions in [the docs](https://doc-sys.psibase.io/documentation.html).
 
-### Ubuntu 22.04
+#### Build flags
 
-Note: building on Ubuntu 18.04 or 20.04 requires much more prep work; see [docker/ubuntu-1804-builder.Dockerfile](docker/ubuntu-1804-builder.Dockerfile) or [docker/ubuntu-2004-builder.Dockerfile](docker/ubuntu-2004-builder.Dockerfile) for details.
+The following are some of the common/useful build flags you may use to configure CMake that modify the build. Open the build directory and run `cmake -LH` to view a list of all options.
 
-```sh
-sudo apt-get update
-sudo apt-get install -yq    \
-    autoconf                \
-    binaryen                \
-    build-essential         \
-    ccache                  \
-    clang                   \
-    cmake                   \
-    curl                    \
-    git                     \
-    libclang-dev            \
-    libssl-dev              \
-    libtool                 \
-    llvm-dev                \
-    pkg-config              \
-    python-is-python3       \
-    wget
-
-export WASI_SDK_PREFIX=~/work/wasi-sdk-17.0
-export PATH=~/work/node-v16.17.0-linux-x64/bin:~/.cargo/bin:$PATH
-
-cd ~/work
-wget https://boostorg.jfrog.io/artifactory/main/release/1.78.0/source/boost_1_78_0.tar.gz
-tar xf boost_1_78_0.tar.gz
-cd boost_1_78_0
-./bootstrap.sh
-sudo ./b2 --prefix=/usr/local --build-dir=build variant=release --with-chrono --with-date_time \
-     --with-filesystem --with-iostreams --with-log --with-program_options --with-system --with-test install
-
-cd ~/work
-wget https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-17/wasi-sdk-17.0-linux.tar.gz
-tar xf wasi-sdk-17.0-linux.tar.gz
-
-cd ~/work
-wget https://nodejs.org/dist/v16.17.0/node-v16.17.0-linux-x64.tar.xz
-tar xf node-v16.17.0-linux-x64.tar.xz
-npm i -g npm yarn
-
-cd ~/work
-curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
-rustup target add wasm32-wasi
-cargo install mdbook mdbook-linkcheck mdbook-mermaid
-```
+| Flag                           | Example Values    | Description                                                                                          |
+| ------------------------------ | ----------------- | ---------------------------------------------------------------------------------------------------- |
+| BUILD_DEBUG_WASM               | `ON`,`OFF`        | Built wasms should include debug information                                                         |
```

Comment:
I don't think this is accurate. There are two variables `BUILD_DEBUG_WASM` and `BUILD_RELEASE_WASM`, which can both be on or off independently.

Reply:
Added build_release_wasm and changed both descriptions